cmd/fpga_crihook/README.md

```diff
@@ -9,27 +9,27 @@
     * [Building the image](#building-the-image)
 * [Configuring CRI-O](#configuring-cri-o)
 
-# Introduction
+## Introduction
 
 The FPGA CRI-O webhook is one of the components used to add support for Intel FPGA
 devices to Kubernetes.
 
 The FPGA prestart CRI-O hook is triggered by container annotations, such as set by the
-[FPGA device plugin](../fpga_plugin).  It performs discovery of the requested FPGA
+[FPGA device plugin](../fpga_plugin/README.md).  It performs discovery of the requested FPGA
 function bitstream and then programs FPGA devices based on the environment variables
 in the workload description.
 
 The CRI-O prestart hook is only *required* when the
-[FPGA admission webhook](../fpga_admissionwebhook) is configured for orchestration
+[FPGA admission webhook](../fpga_admissionwebhook/README.md) is configured for orchestration
 programmed mode, and is benign (un-used) otherwise.
 
 > **Note:** The fpga CRI-O webhook is usually installed by the same DaemonSet as the
 > FPGA device plugin. If building and installing the CRI-O webhook by hand, it is
 > recommended you reference the
-> [fpga plugin DaemonSet YAML](../../deployments/fpga_plugin/fpga_plugin.yaml) for
+> [fpga plugin DaemonSet YAML](/deployments/fpga_plugin/fpga_plugin.yaml) for
```

Comment:
Is this link really broken? Would it be safer as an absolute path? I'm a bit concerned how it's going to be handled by HTML renderers: would it be always treated as absolute inside the repo, not the website (e.g. https://github.com/deployments/...).

Reply:
In this case, that file doesn't exist in that location and doesn't work in the existing documentation. Absolute paths work for both GitHub and Sphinx. The sphinx-md extension can either upload the files with the docs or link to the source commit in GitHub. I prefer the source commit route, which is configurable.

Reply:
there's only `deployments/fpga_plugin/base/intel-fpga-plugin-daemonset.yaml`

Reply:
Oh, right. The file simply doesn't exist in that path in the master branch. Then what appears in the generated docs then?